app/controllers/coronavirus_form/check_answers_controller.rb

```diff
@@ -22,6 +21,19 @@ def submit
 
 private
 
+  def thank_user
```

Comment:
I'd suggest changing this method name to something like `send_confirmation_email` since I was initially expecting it to render a thank you view 😁